tests/perf/prerequisites/event-bus/setup.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+
+set -e
+set -o pipefail
+
+WORKING_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+export NAMESPACE=event-bus-perf-test
+export CLUSTER_DOMAIN=$(kubectl get gateways.networking.istio.io kyma-gateway \
+                        -n kyma-system -ojsonpath="{.spec.servers[0].hosts[0]}" | sed 's/*//g' )
+export SUBSCRIBER_SERVICE_PATH="subscriber-service"
+export SUBSCRIBER_STATUS_URL="https://${SUBSCRIBER_SERVICE_PATH}${CLUSTER_DOMAIN}/v1/status"
+
+eventing_specs=(
+    namespace.yaml
+    event-subscriber.yaml
+    event-activation.yaml
+    event-subscription.yaml
+)
+
+function wait_for_subscriber_to_be_ready() {
+    echo -n "Waiting for subscriber to be ready..."
+    for i in {1..15}; do #timeout after 30 seconds
+        local response=$(curl -i $SUBSCRIBER_STATUS_URL 2>/dev/null)
+        if [[ $response == *"200"* ]]; then
+            echo -e "\nSubscriber is ready."
+            return 0
+        fi
+        echo -n "."
+        sleep 2
+    done
+    echo -e "\n\n ERROR: Subscriber pod is not ready to listen to events\n"
+    return 1
```

Comment:
Why not use `exit 1` instead to signal that there is an error

Reply:
Ideally we should get a slack notification if the setup of one particular test is failing. Without an exit code !=0 this is hard to detect.

Reply:
Same for other functions

Reply:
done